doc/metrics/weight-of-class.md

```diff
@@ -0,0 +1,7 @@
+# Weight Of Class
+
+Number of **functional** public methods divided by the total number of public methods.
+
+This metric tries to quantify whether the measured class' interface reveals more data than behaviour. Low values (less than 33%) indicate that the class reveals much more data than behaviour, which is a sign of poor encapsulation.
+
+Our definition of **functional method** excludes getters and setters.
```

Comment:
From my point of view this metric is very strange: so basically we measure amount of methods without getters and setters, but if a class will have public fields instead of getters/setters this metric will not be low (but basically, idea is the same - more data than behaviour.